### ISSUE TITLE
关于train_SoftMaskedBert中的HYPER_PARAMS问题

### ISSUE THREAD

**Issue by new-cainiao** [gitabtion/BertBasedCorrectionModels#38]
train_SoftMaskedBert中的HYPER_PARAMS数值，是指关于detection loss 与 correction loss的权重占比吗？
这个数值是应用于CscTrainingModel.training_step下的loss = self.w * outputs[1] + (1 - self.w) * outputs[0]吗？
0.8: detection*0.2 + correction*0.8
我是否可以通过修改该数值达到模型侧重于提高detection的prf

**Comment by gitabtion** [comment]
该权重的确是用来调整两个loss的，修改该数据的确可以在一定程度上调整模型在detection或correction上的侧重，但小范围调整可能影响不会太大。
